r/tests/testthat/test-dplyr-string-functions.R

```diff
@@ -719,6 +719,69 @@ test_that("errors in strptime", {
   )
 })
 
+test_that("strftime", {
+  skip_on_os("windows") # https://issues.apache.org/jira/browse/ARROW-13168
+
+  # TODO: consider reevaluating this workaround after ARROW-12980
```

Comment:
Is https://issues.apache.org/jira/browse/ARROW-12980 the issue you're thinking of? Looks like it's been merged now. Is this still relevant?

Reply:
Yeah, I'll try removing this and other such checks ([1](https://github.com/apache/arrow/blob/4678968c1103f0d9d4f86894c0d9a6980bda301e/r/tests/testthat/test-dplyr-string-functions.R#L647), [2](https://github.com/apache/arrow/blob/4678968c1103f0d9d4f86894c0d9a6980bda301e/r/R/dplyr-functions.R#L664))

Reply:
Oh wait no, those are for strptime.